glossary.asciidoc

```diff
@@ -447,7 +447,8 @@ Transaction::
     Transactions can also contain contracts and should not be confused with payments.
 
 Transaction Malleability::
-    TBD.
+    Transaction Malleability is a property that a transaction's hash can change without real changes in transaction content. For example, an unsigned transaction's hash will change after signing, which is inconvenient and may lead to issues. 
```

Comment:
NIT: hash of a transaction...
Formulation: ... without real changes... maybe use without changing the semantic of the transaction? 
Is that really the best example? What about altering the signature which would be the case that is prevented in the channel opening protocol?

Reply:
Hi @renepickhardt  I have make some changes. will be great if you can take a look!